### PR TITLE
rebase upstream

### PR DESCRIPTION
**Pull request by leifmadsen**
- Fix issue when passing common args to the Docker image.
- Allow users to customize the command via ARGS and RUN_ARGS environment variables.
- Update RPM spec and Debian changelog.
- Fix issue in MIB. 'prometheusAlertNotificationRawData' needs to be a 'DisplayString'.
- Add 'Docker' section to README.
- Update README.
- Bump pyyaml from 5.1 to 5.4
- - Improve tox.ini - Fix code
- Add option to enable HTTPS
- Update changelog.
- Add a Dockerfile
- Run dnf clean all after install
- Fix the path and add python3-prometheus_client
- Fedora on Dockerhub still on f31
- Change order of arguments
- Add debugging tools
- Quay has not fedora 32
- Add debug level
- Remove snmp timeout and retries for now
- Update spec file for Fedora
- Fix systemd_pre in spec file
- Address issues from the review
- Adjust Dockerfile to use packages
- Use quay.io as the base image repository (#5)
- migrate ci (#6)
- Enable pip installs and Cachito compatibility (#9)
- Rebase prometheus_webhook_snmp.py against upstream

### CODE REVIEW

**Review by paramite — APPROVED** [review]

File: tox.ini [hunk 1]
```diff
@@ -1,10 +1,48 @@
...
+envlist =
+    fix,
+    lint,
+    test
 skipsdist = true
-sitepackages = true
+
+[base]
+deps =
+    -rrequirements.txt
```
Comment (yadneshk) [inline tox.ini:10]: Shouldn't this be `requirements.txt` ?

File: tox.ini [hunk 2]
```diff
@@ -1,10 +1,48 @@
...
+
+[base-test]
+deps =
+    -rrequirements-test.txt
```
Comment (yadneshk) [inline tox.ini:14]: And this `requirements-test.txt` ?

File: tox.ini (hunk 1 above)
Comment (mrunge) [reply tox.ini:10]: -r is the argument to read from the file given after. I agree, this is hard to read, you see the same e.g. here: https://github.com/openstack/ceilometer/blob/master/tox.ini#L10

**Review by mrunge — APPROVED** [review]
Thank you, looks good to me.

File: tox.ini (hunk 2 above)
Comment (leifmadsen) [reply tox.ini:14]: No such file exists in the upstream repository at https://github.com/SUSE/prometheus-webhook-snmp so I'm not sure this is a problem for us to resolve here.

**Pull request merged**